mlflow/gluon.py

```diff
@@ -222,6 +222,10 @@ def save_model(
             conda_env = yaml.safe_load(f)
     with open(os.path.join(path, conda_env_subpath), "w") as f:
         yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
+
+    pip_req_subpath = "requirements.txt"
+    _log_pip_requirements(conda_env, os.path.join(path, pip_req_subpath))
```

Comment:
It seems like `pip_req_subpath` will also be the same across all the flavors, so we may just want to not make this a parameter in the function and include it in the helper method itself.